vm/kvm-self-tests/runtest.sh

```diff
@@ -61,82 +61,43 @@ function disableTests
 
     # Disable tests for RHEL8 Kernel (4.18.X)
     if uname -r | grep --quiet '^4'; then
-        # Disabled x86_64 tests for Intel & AMD machines due to bugs
-        if [[ $hwpf == "x86_64" ]]; then
-            # Disable test clear_dirty_log_test
-            # due to https://bugzilla.redhat.com/show_bug.cgi?id=1718479
-            mapfile -d $'\0' -t ALLARCH_TESTS < <(printf '%s\0' "${ALLARCH_TESTS[@]}" | grep -Pzv "clear_dirty_log_test")
-
-            # Disable test x86_64/vmx_set_nested_state_test
-            # due to https://bugzilla.redhat.com/show_bug.cgi?id=1740235
-            mapfile -d $'\0' -t X86_64_TESTS < <(printf '%s\0' "${X86_64_TESTS[@]}" | grep -Pzv "x86_64/vmx_set_nested_state_test")
-        fi
-
-        # Disabled x86_64 tests for AMD machines due to bugs
-        if lsmod | grep --quiet kvm_amd; then
-            # Disable test x86_64/platform_info_test
-            # due to https://bugzilla.redhat.com/show_bug.cgi?id=1718499
-            mapfile -d $'\0' -t X86_64_TESTS < <(printf '%s\0' "${X86_64_TESTS[@]}" | grep -Pzv "x86_64/platform_info_test")
-
-            if lscpu | grep --quiet Opteron; then
-                # Disable test x86_64/state_test & x86_64/smm_test
-                # due to https://bugzilla.redhat.com/show_bug.cgi?id=1741347
-                mapfile -d $'\0' -t X86_64_TESTS < <(printf '%s\0' "${X86_64_TESTS[@]}" | grep -Pzv "x86_64/state_test")
-                mapfile -d $'\0' -t X86_64_TESTS < <(printf '%s\0' "${X86_64_TESTS[@]}" | grep -Pzv "x86_64/smm_test")
-            fi
-        fi
+    #    # Disabled x86_64 tests for Intel & AMD machines due to bugs
+    #    if [[ $hwpf == "x86_64" ]]; then
+    #    fi
```

Comment:
I'm not sure I understand this: if this is a "placeholder" so people know where to put their disabled tests? In that case, shouldn't the `if` block itself be uncommented?

Same question about the other two, of course, and another one further down for ARK Kernel.